app/src/NativeBase/Components/Project/ProjectFullDetails.tsx

```diff
@@ -39,7 +39,7 @@ const ProjectFullDetails = ({
   return (
     <>
       <ScrollView
-        _dark={{ backgroundColor: 'bgDarkMode.100' }}
+        _dark={{ backgroundColor: 'bgDarkMode.200' }}
```

Comment:
Could we actually remove these two lines here (42 and 43), and remove the `ScrollView` default background colour settings in `StaTheme.tsx`?  

I can't see any difference between the background colour being used here and the overall app background colours set in `navigationTheme` in `Application.tsx` (and so the `ScrollView` can just be transparent) -- except that there the light background is set to white, and I think that's an accessibility no-no, I'm going to check with the team on Slack.

Reply:
Happy to change this if required.

Reply:
Thanks @kelsiesmurphy just had confirmation from Joanna on the Jira ticket.  

So I think we should change `bg.100` in `StaTheme` to `offWhite100` and remove `bg.secondary` (as it would be the same).  And remove the `ScrollView` default background colours there.  And then you can remove these two lines in this file too.

Reply:
(And hopefully my Friday brain hasn't missed anything / suggested anything daft there!)

Reply:
This is done, over two [commits](https://github.com/Scottish-Tech-Army/Volunteer-app/pull/155/commits/09edb9ebb7f60361bea0082ffc907bd9cd99c45a). I've added a longer form comment below, but I did test this, and I can't see any areas where that secondary bg value was being referenced in the app, but since it is now removed there's a possibility somewhere is sneakily referencing it.